README.md

```diff
@@ -20,10 +20,10 @@ This project is made possible by a [community of contributors](https://github.co
 
 ```bash
 npm install --save ts-json-schema-generator
-./node_modules/.bin/ts-json-schema-generator --path 'my/project/**/*.ts' --type 'My.Type.Full.Name'
+./node_modules/.bin/ts-json-schema-generator --path 'my/project/**/*.ts' --type 'My.Type.Selector'
```

Comment:
Let's do this in a separate pull request.

Reply:
This change specifically or all the changes I've made to the README?

Reply:
Just the changes related to supporting selectors in addition to types.

Reply:
Removed the changes in b6e3c0d6, will submit a separate PR once this is merged with the documentation changes